src/Core/src/PropertyMapperExtensions.cs

```diff
@@ -12,7 +12,8 @@ public static class PropertyMapperExtensions
 		/// <param name="propertyMapper">The property mapper in which to change the mapping.</param>
 		/// <param name="key">The name of the property.</param>
 		/// <param name="method">The modified method to call when the property is updated.</param>
-		public static void ModifyMapping<TVirtualView, TViewHandler>(this IPropertyMapper<TVirtualView, TViewHandler> propertyMapper,
+#pragma warning disable RS0016 // Add public types and members to the declared API
+		public static IPropertyMapper<TVirtualView, TViewHandler> ModifyMapping<TVirtualView, TViewHandler>(this IPropertyMapper<TVirtualView, TViewHandler> propertyMapper,
```

Comment:
I wonder if this is breaking... I think methods can't be overloaded just by return type, so this may actually work. I'll test and report back.